tests/svtypes/global_member_type.ys

```diff
@@ -0,0 +1,21 @@
+read_verilog -sv <<EOF
+typedef logic S;
+EOF
+
+read_verilog -sv <<EOF
+module first;
+	typedef struct packed {
+		S member;
+	} mystruct;
+endmodule
+EOF
+
+read_verilog -sv <<EOF
+module second;
+	typedef logic T;
+	typedef union {
+		S s;
+		T t;
+	} myunion;
```

Comment:
I'd be interested in seeing coverage for typedefs in more deeply nested scopes, i.e., within generate blocks. The changes seem reasonable, but some extra coverage will help us avoid regressions.

Reply:
I added a struct member type test to `typedef_scopes.sv`.